modules/serverless-logic-install-kn-workflow-artifact-images.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-artifact-images_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the artifacts image
+:context: serverless-logic-install-kn-workflow-artifact-images
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
```

Comment:
IDK what is the standard you use here in general, but, from the perspective of the solely installation of the kn-workflow plugin with this new procedure, you don't access to any project. Users can even install the plugin to "play" locally and never connect to the OpenShift

Reply:
removed this prereq